docs/how-to-guides/themes/theme-json.md

```diff
@@ -710,12 +731,16 @@ Within this field themes can list the template parts present in the `block-templ
 
 Currently block variations exist for "header" and "footer" values of the area term, any other values and template parts not defined in the json will default to the general template part block. Variations will be denoted by specific icons within the editor's interface, will default to the corresponding semantic HTML element for the wrapper (this can also be overridden by the `tagName` attribute set on the template part block), and will contextualize the template part allowing more custom flows in future editor improvements.
 
+- name: mandatory.
+- area: optional, will be set to `uncategorized` by default and trigger no block variation.
+
 ```json
 {
+    "version": 1,
 	"templateParts": [
 		{
-			"name": "my-template-part" /* Mandatory */,
-			"area": "header" /* Optional, will be set to 'uncategorized' by default and trigger no block variation */
+			"name": "my-template-part",
+			"area": "header"
 		}
 	]
```

Comment:
There's one I didn't change because it's an example of something that shouldn't be done, so I've left it intentionally that way. I can attach a comment in that place, so this is the example for reference:

```json
{
	"version": 1,
	"settings": {
		"custom": {
			"line--height": { // DO NOT DO THIS
				"body": 1.7
			}
		}
	}
}
```